test/log.ts

```diff
@@ -187,32 +188,30 @@ describe('Log', () => {
   });
 
   describe('delete', () => {
-    it('should accept gaxOptions', done => {
-      log.logging.request = (config, callback) => {
-        assert.strictEqual(config.client, 'LoggingServiceV2Client');
-        assert.strictEqual(config.method, 'deleteLog');
+    beforeEach(() => {
+      log.logging.auth.getProjectId = async () => PROJECT_ID;
+    });
 
-        assert.deepStrictEqual(config.reqOpts, {
+    it('should execute gax method', async () => {
+      log.logging.loggingService.deleteLog = async (reqOpts, gaxOpts) => {
+        assert.deepStrictEqual(reqOpts, {
           logName: log.formattedName_,
         });
 
-        assert.deepStrictEqual(config.gaxOpts, {});
-
-        callback(); // done()
+        assert.strictEqual(gaxOpts, undefined);
       };
 
-      log.delete(done);
+      await log.delete();
     });
 
-    it('should accept gaxOptions', done => {
+    it('should accept gaxOptions', async () => {
       const gaxOptions = {};
 
-      log.logging.request = config => {
-        assert.strictEqual(config.gaxOpts, gaxOptions);
-        done();
+      log.logging.loggingService.deleteLog = async (reqOpts, gaxOpts) => {
+        assert.deepStrictEqual(gaxOpts, gaxOptions);
```

Comment:
Does `deepStrictEqual` also check `gaxOpts === gaxOptions`? If it's just checking the properties, this probably isn't doing what we want, since we aren't setting any properties inside of the `gaxOptions` object.

Reply:
Switched to `stricktEqual`